app/tracingStrategy.ts

```diff
@@ -13,11 +15,11 @@ export interface TracingStrategy {
 }
 
 export interface StrategyAssets {
-  personalPrivacyBackground: string;
+  personalPrivacyBackground: ImageSourcePropType;
```

Comment:
This is the correct type for an image source. [See React Native docs.](https://reactnative.dev/docs/0.32/image#source)